### DISCUSSION TITLE
Add support to Pascal fenced codeblocks

### DISCUSSION THREAD

**Discussion by fabvit** [Zettlr/Zettlr#5370] [Polls]
I think that it would be useful (to Pascal programmers) to add support for the Pascal programmin language in Zettlr's fenced codeblocks.

**Comment by lordofscripts** [comment]
Wow! Pascal! I loved that language! I learned it after Basic & C. Turbo Pascal was like a dream those days. I tried that free pascal a few weeks ago but it was kind of very old tooling. Is there a better way of doing Pascal these days with VSCode?

**Comment by benniekiss** [comment]
This should be a very simple addition. Codemirror already provides the language support in https://github.com/codemirror/legacy-modes

**Comment by nathanlesage** [comment]
No work needed, Zettlr has Pascal support already, keyword `pascal`:

<img width="610" height="79" alt="image" src="https://github.com/user-attachments/assets/02341cfc-06f4-4765-b477-a0d34f2dc87d" />

A quick consultation of the documentation shows all available code languages so far: https://docs.zettlr.com/en/scientific-technical/code-blocks/#available-languages

**Reply by benniekiss** [reply]
heh, I did not realize how old the OP was!
